src/geop/main.py

```diff
@@ -1,11 +1,60 @@
-from flask import Flask
+import os
+import time
+
+from flask import Flask, request, jsonify
+import numpy as np
+import rasterio
 
 app = Flask(__name__)
 
+DATA_PATH = '/usr/data/'
+
+
+@app.route('/counts', methods=['POST'])
+def count():
+    """
+    Perform a rudimentary cell count analysis on a portion of a
+    provided raster
+
+    Query String Args:
+        filename: file name of a valid geotiff in DATA_DIR
+    """
+
+    raster = request.args.get('filename')
+    if not raster:
+        return handle_error('filename parameter is required')
+
+    raster_path = os.path.join(DATA_PATH, raster)
+    if not os.path.isfile(raster_path):
+        return handle_error('{} is not valid file in DATA_DIR'.format(raster))
+
+    counts = {}
```

Comment:
May be able to use a `defaultdict(int)` here to simplify the logic within the `for` loop for `cell_val`:

```python
>>> from collections import defaultdict
>>> x = defaultdict(int)
>>> x['a'] += 1
>>> x
defaultdict(<type 'int'>, {'a': 1})
```